homeassistant/components/light/zwave.py

```diff
@@ -244,9 +235,9 @@ def _value_added(self, value):
         # Check for the missing color values
         self._get_color_values()
 
-    def update_properties(self):
+    def update(self):
```

Comment:
Just for completeness of discussion, this is what would happen if someone would change the order of inheritence, ie `Device(Light, ZWaveDeviceEntity):`, instead of the current `Device(ZWaveDeviceEntity, Light):`.

```py
In [42]: %paste
class Entity(object):
    """docstring for Entity"""
    def update(self):
        print('{} update'.format(Entity.__name__))

class Light(Entity):
    """docstring for Light"""
    # def update(self):
    # print('{} update'.format(Light.__name__))

class ZWaveDevice(Entity):
    """docstring for ZWaveDevice"""
    def update(self):
        print('{} update'.format(ZWaveDevice.__name__))

class Device(Light, ZWaveDevice):
    """docstring for Device"""
    # def update(self):
    # print('{} update'.format(Device.__name__))
## -- End pasted text --

In [43]: test_entity = Device()

In [44]: test_entity.update()
ZWaveDevice update
```
Remember this is for python3. Python2 has a different MRO profile, ie it doesn't use the last rule "good head". I liked this blog:
https://makina-corpus.com/blog/metier/2014/python-tutorial-understanding-python-mro-class-search-path

If we would both have this new order of inheritence and `Light` would also overwrite `update` we would get this:

```py
In [45]: %paste
class Entity(object):
    """docstring for Entity"""
    def update(self):
        print('{} update'.format(Entity.__name__))

class Light(Entity):
    """docstring for Light"""
    def update(self):
        print('{} update'.format(Light.__name__))

class ZWaveDevice(Entity):
    """docstring for ZWaveDevice"""
    def update(self):
        print('{} update'.format(ZWaveDevice.__name__))

class Device(Light, ZWaveDevice):
    """docstring for Device"""
    # def update(self):
    # print('{} update'.format(Device.__name__))
## -- End pasted text --

In [46]: test_entity = Device()

In [47]: test_entity.update()
Light update
```